modules/RTC/ScreenObtainer.js

```diff
@@ -208,23 +211,28 @@ var ScreenObtainer = {
                         }, 500);
                     },
                     function (arg) {
-                        logger.log("Failed to install the extension from:"
-                            + getWebStoreInstallUrl(self.options), arg);
-                        failCallback({
-                            type: "jitsiError",
-                            errorObject: JitsiTrackErrors
-                                .CHROME_EXTENSION_INSTALLATION_ERROR
-                        });
+                        var msg = "Failed to install the extension from "
+                            + getWebStoreInstallUrl(self.options);
+
+                        logger.log(msg, arg);
+
+                        failCallback(new JitsiTrackError(
+                            JitsiTrackErrors
+                                .CHROME_EXTENSION_INSTALLATION_ERROR,
+                            msg
+                        ));
                     }
                 );
             } catch(e) {
-                logger.log("Failed to install the extension from:"
-                    + self.getWebStoreInstallUrl(this.options), e);
-                failCallback({
-                    type: "jitsiError",
-                    errorObject:
-                        JitsiTrackErrors.CHROME_EXTENSION_INSTALLATION_ERROR
-                });
+                var msg = "Failed to install the extension from "
+                    + getWebStoreInstallUrl(self.options);
+
+                logger.log(msg, e);
+
+                failCallback(new JitsiTrackError(
+                    JitsiTrackErrors.CHROME_EXTENSION_INSTALLATION_ERROR,
+                    msg
+                ));
```

Comment:
Aren't these 9 lines the same as the 9 lines that are 3 lines above them and that are conveniently wrapped into an anonymous function?